config/end2end.js

```diff
@@ -10,9 +10,9 @@ module.exports = {
       disableUpload: true,
     },
   },
-  elife: {
+  server: {
     api: {
-      url: 'https://end2end--gateway.elifesciences.org/',
+      url: 'http://end2end--gateway.elife.internal',
```

Comment:
some of the `server.api.url` values have a trailing `/`, some haven't, not sure if this is alright

Reply:
Ooo - good spot - the tests did work but I think I will change them just to make sure!